template/index.html

```diff
@@ -383,6 +383,16 @@ <h3>{{__ "Parameters"}}</h3>
               <div class="form-group">
                 <label class="col-md-3 control-label" for="sample-request-param-field-{{field}}">{{field}}</label>
                 <div class="input-group">
+                  {{#if allowedValues}}
+                  <div class="input-group-addon sample-request-select">
+                    <select>
+                      <option value=""></option>
```

Comment:
Why put an empty option here? If we must select from a list of allowed values, this is not needed.

Reply:
@NicolasCARPi It make sense if the parameter is optional.

Reply:
I see, then add it after the loop on values, and maybe instead of being empty, name it  `null` to make it a bit more clear that this means we don't send anything with this parameter.

Reply:
Done.